qt/aqt/reviewer.py

```diff
@@ -591,6 +591,20 @@ def korean_shortcuts(
     def _shortcutKeys(
         self,
     ) -> Sequence[tuple[str, Callable] | tuple[Qt.Key, Callable]]:
+
+        def generate_default_answer_keys():
+            for ease in aqt.mw.pm.default_answer_keys:
+                key = aqt.mw.pm.get_answer_key(ease)
+                if key:
+                    if ease in (1, 2, 3, 4):
```

Comment:
We control default_answer_keys, so there should be no need to check this.

Reply:
I'll remove the check then. 👍